keyboards/2_milk/keymaps/emoji/keymap.c

```diff
@@ -0,0 +1,8 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+      [0] = LAYOUT(
+          send_unicode_hex_string("00AF 005C 005F 0028 30C4 0029 005F 002F 00AF"),
```

Comment:
I don't think that you can do this.  I think you need to add a custom keycode to handle this. 

https://docs.qmk.fm/#/feature_macros

Reply:
Thanks I'll do that

Reply:
Does 6326a66 fix it?